### ISSUE TITLE
Plasma freezes under Linux 5.10.77

### ISSUE THREAD

**Issue by Nekdo** [NixOS/nixpkgs#145601]
### Describe the bug

Plasma desktop is unusable.

### Steps To Reproduce
1. Login to plasma
2. Click bottom left to start application launcher
3. Plasma freezes

### Expected behavior
No freezes should happen

### Additional context

This is caused by the regression in linux kernel 5.10.77, see e.g. [this bugreport](https://gitlab.freedesktop.org/drm/amd/-/issues/1778) that also describes my problem.

I started seeing error messages from kernel like this:
```
lis 09 00:39:27 honzk kernel: kernel tried to execute NX-protected page - exploit attempt? (uid: 1000)
lis 09 00:39:27 honzk kernel: BUG: unable to handle page fault for address: ffff9d1dc3a0f800
lis 09 00:39:27 honzk kernel: #PF: supervisor instruction fetch in kernel mode
lis 09 00:39:27 honzk kernel: #PF: error_code(0x0011) - permissions violation
lis 09 00:39:27 honzk kernel: PGD 671401067 P4D 671401067 PUD 83efff067 PMD 104160063 PTE 8000000103a0f163
lis 09 00:39:27 honzk kernel: Oops: 0011 [#1] SMP NOPTI
lis 09 00:39:27 honzk kernel: CPU: 3 PID: 2006 Comm: kwin_x11:rcs0 Not tainted 5.10.77 #1-NixOS
lis 09 00:39:27 honzk kernel: Hardware name: To be filled by O.E.M. To be filled by O.E.M./M5A97 LE R2.0, BIOS 1903 07/11/2013
lis 09 00:39:27 honzk kernel: RIP: 0010:0xffff9d1dc3a0f800
lis 09 00:39:27 honzk kernel: Code: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 <08> e1 16 c2 1d 9d ff ff 08 e1 16 c2 1d 9d ff ff 00 00 00 00 00 00
lis 09 00:39:27 honzk kernel: RSP: 0018:ffffc04bc330fad8 EFLAGS: 00010286
lis 09 00:39:27 honzk kernel: RAX: ffff9d1dc3a0f800 RBX: 0000000000000001 RCX: 0000000000000001
lis 09 00:39:27 honzk kernel: RDX: 7fffffffffffffff RSI: 0000000000000001 RDI: ffff9d1dc2272800
lis 09 00:39:27 honzk kernel: RBP: ffff9d1dc2272800 R08: 0000000000000000 R09: 0000000000000000
lis 09 00:39:27 honzk kernel: R10: 0000000000000000 R11: 0000000000000410 R12: 7fffffffffffffff
lis 09 00:39:27 honzk kernel: R13: 0000000000000001 R14: ffff9d1dc30d0018 R15: ffff9d1dd699ce00
lis 09 00:39:27 honzk kernel: FS:  00007f0a5cdc7640(0000) GS:ffff9d24decc0000(0000) knlGS:0000000000000000
lis 09 00:39:27 honzk kernel: CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
lis 09 00:39:27 honzk kernel: CR2: ffff9d1dc3a0f800 CR3: 000000013f856000 CR4: 00000000000406e0
lis 09 00:39:27 honzk kernel: Call Trace:
lis 09 00:39:27 honzk kernel:  ? dma_fence_wait_timeout+0x3f/0xd0
lis 09 00:39:27 honzk kernel:  ? radeon_sync_resv+0x40/0x110 [radeon]
lis 09 00:39:27 honzk kernel:  ? radeon_cs_sync_rings+0x55/0x70 [radeon]
lis 09 00:39:27 honzk kernel:  ? radeon_cs_ioctl+0x610/0x760 [radeon]
lis 09 00:39:27 honzk kernel:  ? radeon_cs_parser_init+0x20/0x20 [radeon]
lis 09 00:39:27 honzk kernel:  ? drm_ioctl_kernel+0xaa/0xf0 [drm]
lis 09 00:39:27 honzk kernel:  ? drm_ioctl+0x201/0x3b0 [drm]
lis 09 00:39:27 honzk kernel:  ? radeon_cs_parser_init+0x20/0x20 [radeon]
lis 09 00:39:27 honzk kernel:  ? selinux_file_ioctl+0x135/0x230
lis 09 00:39:27 honzk kernel:  ? radeon_drm_ioctl+0x49/0x80 [radeon]
lis 09 00:39:27 honzk kernel:  ? __x64_sys_ioctl+0x83/0xb0
lis 09 00:39:27 honzk kernel:  ? do_syscall_64+0x33/0x40
lis 09 00:39:27 honzk kernel:  ? entry_SYSCALL_64_after_hwframe+0x44/0xa9
lis 09 00:39:27 honzk kernel: Modules linked in: udp_diag tcp_diag inet_diag af_packet amdgpu iommu_v2 gpu_sched xt_CHECKSUM xt_MASQUERADE ipt_REJECT nf_reject_ipv4 ip6table_mangle iptable_mangle cfg80211 8021q msr ext4 crc16 mbcache jbd2 nls_iso8859_1 nls_cp437 vfat fat ip6table_nat iptable_nat nf_nat xt_conntrack nf_conntrack nf_defrag_ipv6 nf_defrag_ipv4 ip6t_rpfilter ipt_rpfilter ip6table_raw iptable_raw joydev input_leds mousedev xt_pkttype nf_log_ipv6 nf_log_ipv4 nf_log_common xt_LOG eeepc_wmi xt_tcpudp snd_hda_codec_realtek asus_wmi snd_hda_codec_generic battery sparse_keymap radeon ledtrig_audio snd_hda_codec_hdmi ip6table_filter ip6_tables rfkill snd_hda_intel amd64_edac_mod snd_intel_dspcfg iptable_filter sch_fq_codel edac_mce_amd video hid_generic edac_core uinput snd_hda_codec led_class wmi_bmof atkbd mxm_wmi evdev ttm libps2 crc32_pclmul mac_hid serio ghash_clmulni_intel snd_hda_core snd_hwdep drm_kms_helper snd_pcm aesni_intel deflate usbhid r8169 sp5100_tco libaes i2c_algo_bit watchdog
lis 09 00:39:27 honzk kernel:  snd_timer crypto_simd realtek fb_sys_fops cryptd snd glue_helper efi_pstore k10temp syscopyarea mdio_devres fam15h_power sd_mod sysfillrect i2c_piix4 libphy soundcore sysimgblt hid tiny_power_button wmi acpi_cpufreq button loop cpufreq_ondemand tap macvlan bridge stp llc tun kvm_amd kvm drm irqbypass agpgart fuse backlight pstore i2c_core configfs efivarfs ip_tables x_tables autofs4 ohci_pci xhci_pci xhci_pci_renesas ahci nvme libahci xhci_hcd ehci_pci ohci_hcd libata nvme_core ehci_hcd t10_pi crc_t10dif usbcore crct10dif_generic scsi_mod crct10dif_pclmul crct10dif_common usb_common rtc_cmos btrfs blake2b_generic libcrc32c crc32c_generic crc32c_intel xor raid6_pq dm_snapshot dm_bufio dm_mod
lis 09 00:39:27 honzk kernel: CR2: ffff9d1dc3a0f800
lis 09 00:39:27 honzk kernel: ---[ end trace 8dcccfb686f3b05d ]---
lis 09 00:39:27 honzk kernel: RIP: 0010:0xffff9d1dc3a0f800
lis 09 00:39:27 honzk kernel: Code: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 <08> e1 16 c2 1d 9d ff ff 08 e1 16 c2 1d 9d ff ff 00 00 00 00 00 00
lis 09 00:39:27 honzk kernel: RSP: 0018:ffffc04bc330fad8 EFLAGS: 00010286
lis 09 00:39:27 honzk kernel: RAX: ffff9d1dc3a0f800 RBX: 0000000000000001 RCX: 0000000000000001
lis 09 00:39:27 honzk kernel: RDX: 7fffffffffffffff RSI: 0000000000000001 RDI: ffff9d1dc2272800
lis 09 00:39:27 honzk kernel: RBP: ffff9d1dc2272800 R08: 0000000000000000 R09: 0000000000000000
lis 09 00:39:27 honzk kernel: R10: 0000000000000000 R11: 0000000000000410 R12: 7fffffffffffffff
lis 09 00:39:27 honzk kernel: R13: 0000000000000001 R14: ffff9d1dc30d0018 R15: ffff9d1dd699ce00
lis 09 00:39:27 honzk kernel: FS:  00007f0a5cdc7640(0000) GS:ffff9d24decc0000(0000) knlGS:0000000000000000
lis 09 00:39:27 honzk kernel: CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
lis 09 00:39:27 honzk kernel: CR2: ffff9d1dc3a0f800 CR3: 000000013f856000 CR4: 00000000000406e0
lis 09 00:39:27 honzk kernel: ------------[ cut here ]------------
lis 09 00:39:27 honzk kernel: [Firmware Bug]: Page fault caused by firmware at PA: 0xffff9d1dc3a0f800
lis 09 00:39:27 honzk kernel: WARNING: CPU: 0 PID: 2101 at arch/x86/platform/efi/quirks.c:712 efi_recover_from_page_fault+0x2a/0xc0
lis 09 00:39:27 honzk kernel: Modules linked in: udp_diag tcp_diag inet_diag af_packet amdgpu iommu_v2 gpu_sched xt_CHECKSUM xt_MASQUERADE ipt_REJECT nf_reject_ipv4 ip6table_mangle iptable_mangle cfg80211 8021q msr ext4 crc16 mbcache jbd2 nls_iso8859_1 nls_cp437 vfat fat ip6table_nat iptable_nat nf_nat xt_conntrack nf_conntrack nf_defrag_ipv6 nf_defrag_ipv4 ip6t_rpfilter ipt_rpfilter ip6table_raw iptable_raw joydev input_leds mousedev xt_pkttype nf_log_ipv6 nf_log_ipv4 nf_log_common xt_LOG eeepc_wmi xt_tcpudp snd_hda_codec_realtek asus_wmi snd_hda_codec_generic battery sparse_keymap radeon ledtrig_audio snd_hda_codec_hdmi ip6table_filter ip6_tables rfkill snd_hda_intel amd64_edac_mod snd_intel_dspcfg iptable_filter sch_fq_codel edac_mce_amd video hid_generic edac_core uinput snd_hda_codec led_class wmi_bmof atkbd mxm_wmi evdev ttm libps2 crc32_pclmul mac_hid serio ghash_clmulni_intel snd_hda_core snd_hwdep drm_kms_helper snd_pcm aesni_intel deflate usbhid r8169 sp5100_tco libaes i2c_algo_bit watchdog
lis 09 00:39:27 honzk kernel:  snd_timer crypto_simd realtek fb_sys_fops cryptd snd glue_helper efi_pstore k10temp syscopyarea mdio_devres fam15h_power sd_mod sysfillrect i2c_piix4 libphy soundcore sysimgblt hid tiny_power_button wmi acpi_cpufreq button loop cpufreq_ondemand tap macvlan bridge stp llc tun kvm_amd kvm drm irqbypass agpgart fuse backlight pstore i2c_core configfs efivarfs ip_tables x_tables autofs4 ohci_pci xhci_pci xhci_pci_renesas ahci nvme libahci xhci_hcd ehci_pci ohci_hcd libata nvme_core ehci_hcd t10_pi crc_t10dif usbcore crct10dif_generic scsi_mod crct10dif_pclmul crct10dif_common usb_common rtc_cmos btrfs blake2b_generic libcrc32c crc32c_generic crc32c_intel xor raid6_pq dm_snapshot dm_bufio dm_mod
lis 09 00:39:27 honzk kernel: CPU: 0 PID: 2101 Comm: plasmashe:rcs0 Tainted: G      D           5.10.77 #1-NixOS
lis 09 00:39:27 honzk kernel: Hardware name: To be filled by O.E.M. To be filled by O.E.M./M5A97 LE R2.0, BIOS 1903 07/11/2013
lis 09 00:39:27 honzk kernel: RIP: 0010:efi_recover_from_page_fault+0x2a/0xc0
lis 09 00:39:27 honzk kernel: Code: 0f 1f 44 00 00 8b 15 35 29 93 01 85 d2 74 09 48 81 ff ff 0f 00 00 77 01 c3 53 48 89 fe 48 c7 c7 a0 b5 b6 b4 50 e8 79 13 78 00 <0f> 0b 83 3d 0d 29 93 01 0a 0f 84 b2 0e 78 00 48 8b 3d f8 43 8b 01
lis 09 00:39:27 honzk kernel: RSP: 0018:ffffc04bc3587980 EFLAGS: 00010082
lis 09 00:39:27 honzk kernel: RAX: 0000000000000000 RBX: ffff9d1e0335ba00 RCX: ffff9d24dec18a88
lis 09 00:39:27 honzk kernel: RDX: 00000000ffffffd8 RSI: 0000000000000027 RDI: ffff9d24dec18a80
lis 09 00:39:27 honzk kernel: RBP: ffffc04bc3587a28 R08: ffffffffb4f381e8 R09: 0000000000009ffb
lis 09 00:39:27 honzk kernel: R10: 00000000ffffe000 R11: 3fffffffffffffff R12: ffff9d1dc3a0f800
lis 09 00:39:27 honzk kernel: R13: 0000000000000011 R14: 000000000000000b R15: 0000000000000001
lis 09 00:39:27 honzk kernel: FS:  00007f9cbffff640(0000) GS:ffff9d24dec00000(0000) knlGS:0000000000000000
lis 09 00:39:27 honzk kernel: CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
lis 09 00:39:27 honzk kernel: CR2: ffff9d1dc3a0f800 CR3: 0000000143228000 CR4: 00000000000406f0
lis 09 00:39:27 honzk kernel: Call Trace:
lis 09 00:39:27 honzk kernel:  no_context+0x13c/0x380
lis 09 00:39:27 honzk kernel:  exc_page_fault+0x9c/0x120
lis 09 00:39:27 honzk kernel:  asm_exc_page_fault+0x1e/0x30
lis 09 00:39:27 honzk kernel: RIP: 0010:0xffff9d1dc3a0f800
lis 09 00:39:27 honzk kernel: Code: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 <08> e1 16 c2 1d 9d ff ff 08 e1 16 c2 1d 9d ff ff 00 00 00 00 00 00
lis 09 00:39:27 honzk kernel: RSP: 0018:ffffc04bc3587ad8 EFLAGS: 00010286
lis 09 00:39:27 honzk kernel: RAX: ffff9d1dc3a0f800 RBX: 0000000000000001 RCX: 0000000000000001
lis 09 00:39:27 honzk kernel: RDX: 7fffffffffffffff RSI: 0000000000000001 RDI: ffff9d1e155ac080
lis 09 00:39:27 honzk kernel: RBP: ffff9d1e155ac080 R08: 0000000000000000 R09: 0000000000000000
lis 09 00:39:27 honzk kernel: R10: ffffc04bc0e5e000 R11: ffffc04bc0e5e000 R12: 7fffffffffffffff
lis 09 00:39:27 honzk kernel: R13: 0000000000000001 R14: ffff9d1dc30d0018 R15: ffff9d1de4ed4800
lis 09 00:39:27 honzk kernel:  ? dma_fence_wait_timeout+0x3f/0xd0
lis 09 00:39:27 honzk kernel:  ? radeon_sync_resv+0x40/0x110 [radeon]
lis 09 00:39:27 honzk kernel:  ? radeon_cs_sync_rings+0x55/0x70 [radeon]
lis 09 00:39:27 honzk kernel:  ? radeon_cs_ioctl+0x610/0x760 [radeon]
lis 09 00:39:27 honzk kernel:  ? radeon_cs_parser_init+0x20/0x20 [radeon]
lis 09 00:39:27 honzk kernel:  ? drm_ioctl_kernel+0xaa/0xf0 [drm]
lis 09 00:39:27 honzk kernel:  ? drm_ioctl+0x201/0x3b0 [drm]
lis 09 00:39:27 honzk kernel:  ? radeon_cs_parser_init+0x20/0x20 [radeon]
lis 09 00:39:27 honzk kernel:  ? selinux_file_ioctl+0x135/0x230
lis 09 00:39:27 honzk kernel:  ? radeon_drm_ioctl+0x49/0x80 [radeon]
lis 09 00:39:27 honzk kernel:  ? __x64_sys_ioctl+0x83/0xb0
lis 09 00:39:27 honzk kernel:  ? do_syscall_64+0x33/0x40
lis 09 00:39:27 honzk kernel:  ? entry_SYSCALL_64_after_hwframe+0x44/0xa9
lis 09 00:39:27 honzk kernel: ---[ end trace 8dcccfb686f3b05e ]---
lis 09 00:39:27 honzk kernel: ------------[ cut here ]------------
lis 09 00:39:27 honzk kernel: WARNING: CPU: 0 PID: 2101 at arch/x86/kernel/fpu/core.c:155 kernel_fpu_end+0x19/0x20
lis 09 00:39:27 honzk kernel: Modules linked in: udp_diag tcp_diag inet_diag af_packet amdgpu iommu_v2 gpu_sched xt_CHECKSUM xt_MASQUERADE ipt_REJECT nf_reject_ipv4 ip6table_mangle iptable_mangle cfg80211 8021q msr ext4 crc16 mbcache jbd2 nls_iso8859_1 nls_cp437 vfat fat ip6table_nat iptable_nat nf_nat xt_conntrack nf_conntrack nf_defrag_ipv6 nf_defrag_ipv4 ip6t_rpfilter ipt_rpfilter ip6table_raw iptable_raw joydev input_leds mousedev xt_pkttype nf_log_ipv6 nf_log_ipv4 nf_log_common xt_LOG eeepc_wmi xt_tcpudp snd_hda_codec_realtek asus_wmi snd_hda_codec_generic battery sparse_keymap radeon ledtrig_audio snd_hda_codec_hdmi ip6table_filter ip6_tables rfkill snd_hda_intel amd64_edac_mod snd_intel_dspcfg iptable_filter sch_fq_codel edac_mce_amd video hid_generic edac_core uinput snd_hda_codec led_class wmi_bmof atkbd mxm_wmi evdev ttm libps2 crc32_pclmul mac_hid serio ghash_clmulni_intel snd_hda_core snd_hwdep drm_kms_helper snd_pcm aesni_intel deflate usbhid r8169 sp5100_tco libaes i2c_algo_bit watchdog
lis 09 00:39:27 honzk kernel:  snd_timer crypto_simd realtek fb_sys_fops cryptd snd glue_helper efi_pstore k10temp syscopyarea mdio_devres fam15h_power sd_mod sysfillrect i2c_piix4 libphy soundcore sysimgblt hid tiny_power_button wmi acpi_cpufreq button loop cpufreq_ondemand tap macvlan bridge stp llc tun kvm_amd kvm drm irqbypass agpgart fuse backlight pstore i2c_core configfs efivarfs ip_tables x_tables autofs4 ohci_pci xhci_pci xhci_pci_renesas ahci nvme libahci xhci_hcd ehci_pci ohci_hcd libata nvme_core ehci_hcd t10_pi crc_t10dif usbcore crct10dif_generic scsi_mod crct10dif_pclmul crct10dif_common usb_common rtc_cmos btrfs blake2b_generic libcrc32c crc32c_generic crc32c_intel xor raid6_pq dm_snapshot dm_bufio dm_mod
lis 09 00:39:27 honzk kernel: CPU: 0 PID: 2101 Comm: plasmashe:rcs0 Tainted: G      D W         5.10.77 #1-NixOS
lis 09 00:39:27 honzk kernel: Hardware name: To be filled by O.E.M. To be filled by O.E.M./M5A97 LE R2.0, BIOS 1903 07/11/2013
lis 09 00:39:27 honzk kernel: RIP: 0010:kernel_fpu_end+0x19/0x20
lis 09 00:39:27 honzk kernel: Code: ae 47 40 b8 01 00 00 00 c3 0f 0b eb d7 0f 0b eb c9 0f 1f 44 00 00 65 8a 05 7c 1c 3e 4c 84 c0 74 09 65 c6 05 70 1c 3e 4c 00 c3 <0f> 0b eb f3 0f 1f 00 0f 1f 44 00 00 8b 15 b5 6c 8d 01 31 f6 e8 ae
lis 09 00:39:27 honzk kernel: RSP: 0018:ffffc04bc3587978 EFLAGS: 00010046
lis 09 00:39:27 honzk kernel: RAX: 0000000000000000 RBX: ffff9d1e0335ba00 RCX: 0000000000000048
lis 09 00:39:27 honzk kernel: RDX: 0000000000000000 RSI: ffff9d1dc3bb3fc0 RDI: 0000000000000046
lis 09 00:39:27 honzk kernel: RBP: ffffc04bc3587a28 R08: 0000000000000000 R09: 0000000000000001
lis 09 00:39:27 honzk kernel: R10: 00000000ffffe000 R11: 3fffffffffffffff R12: ffff9d1dc3a0f800
lis 09 00:39:27 honzk kernel: R13: 0000000000000011 R14: 000000000000000b R15: 0000000000000001
lis 09 00:39:27 honzk kernel: FS:  00007f9cbffff640(0000) GS:ffff9d24dec00000(0000) knlGS:0000000000000000
lis 09 00:39:27 honzk kernel: CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
lis 09 00:39:27 honzk kernel: CR2: ffff9d1dc3a0f800 CR3: 00000001082d4000 CR4: 00000000000406f0
lis 09 00:39:27 honzk kernel: Call Trace:
lis 09 00:39:27 honzk kernel:  efi_recover_from_page_fault+0x5f/0xc0
lis 09 00:39:27 honzk kernel:  no_context+0x13c/0x380
lis 09 00:39:27 honzk kernel:  exc_page_fault+0x9c/0x120
lis 09 00:39:27 honzk kernel:  asm_exc_page_fault+0x1e/0x30
lis 09 00:39:27 honzk kernel: RIP: 0010:0xffff9d1dc3a0f800
lis 09 00:39:27 honzk kernel: Code: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 <08> e1 16 c2 1d 9d ff ff 08 e1 16 c2 1d 9d ff ff 00 00 00 00 00 00
lis 09 00:39:27 honzk kernel: RSP: 0018:ffffc04bc3587ad8 EFLAGS: 00010286
lis 09 00:39:27 honzk kernel: RAX: ffff9d1dc3a0f800 RBX: 0000000000000001 RCX: 0000000000000001
lis 09 00:39:27 honzk kernel: RDX: 7fffffffffffffff RSI: 0000000000000001 RDI: ffff9d1e155ac080
lis 09 00:39:27 honzk kernel: RBP: ffff9d1e155ac080 R08: 0000000000000000 R09: 0000000000000000
lis 09 00:39:27 honzk kernel: R10: ffffc04bc0e5e000 R11: ffffc04bc0e5e000 R12: 7fffffffffffffff
lis 09 00:39:27 honzk kernel: R13: 0000000000000001 R14: ffff9d1dc30d0018 R15: ffff9d1de4ed4800
lis 09 00:39:27 honzk kernel:  ? dma_fence_wait_timeout+0x3f/0xd0
lis 09 00:39:27 honzk kernel:  ? radeon_sync_resv+0x40/0x110 [radeon]
lis 09 00:39:27 honzk kernel:  ? radeon_cs_sync_rings+0x55/0x70 [radeon]
lis 09 00:39:27 honzk kernel:  ? radeon_cs_ioctl+0x610/0x760 [radeon]
lis 09 00:39:27 honzk kernel:  ? radeon_cs_parser_init+0x20/0x20 [radeon]
lis 09 00:39:27 honzk kernel:  ? drm_ioctl_kernel+0xaa/0xf0 [drm]
lis 09 00:39:27 honzk kernel:  ? drm_ioctl+0x201/0x3b0 [drm]
lis 09 00:39:27 honzk kernel:  ? radeon_cs_parser_init+0x20/0x20 [radeon]
lis 09 00:39:27 honzk kernel:  ? selinux_file_ioctl+0x135/0x230
lis 09 00:39:27 honzk kernel:  ? radeon_drm_ioctl+0x49/0x80 [radeon]
lis 09 00:39:27 honzk kernel:  ? __x64_sys_ioctl+0x83/0xb0
lis 09 00:39:27 honzk kernel:  ? do_syscall_64+0x33/0x40
lis 09 00:39:27 honzk kernel:  ? entry_SYSCALL_64_after_hwframe+0x44/0xa9
lis 09 00:39:27 honzk kernel: ---[ end trace 8dcccfb686f3b05f ]---
```

Reverting kernel to 5.10.76 fixes the problem.
The bug seems to be fixed in 5.10.78, so maybe just update to the newer version will fix the problem.

### Notify maintainers

@TredwellGit
@Ma27
@NeQuissimus 
@alyssais
@thoughtpolice 

### Metadata
```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.76, NixOS, 21.05.20211108.e748941 (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20211006_53e4794`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```
Note: I'm running the older kernel, so I'm able to run browser and report the issue. The issue is occurring with Linux 5.10.77

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: `linuxPackages.kernel` ?

# a list of nixos modules affected by the problem
module: ?
```

**Comment by alyssais** [comment]
> The bug seems to be fixed in 5.10.78, so maybe just update to the newer version will fix the problem.

5.10.78 is now in unstable — is this fixed no?

**Comment by TredwellGit** [comment]
https://nixpk.gs/pr-tracker.html?pr=145358
